include/stateMachineFwk/smEnumTypes.h

```diff
@@ -177,6 +177,7 @@ static constexpr const char* Events[] =
 
 enum Action_e
 {
+    ABORT_ATTACH,
```

Comment:
Good practice to start with 1...since default values of allocation might be 0 and uninitialized variables would cause problems.

Reply:
Created cleanup task for this. Planning to change the enum type to integral type so that state machine framework does not have to depend on the header containing application content.Will fix this in the next iteration.